lasagne/layers/normalization.py

```diff
@@ -365,8 +365,11 @@ def batch_norm(layer, **kwargs):
     if hasattr(layer, 'b') and layer.b is not None:
         del layer.params[layer.b]
         layer.b = None
-    layer = BatchNormLayer(layer, **kwargs)
+    bn_name = kwargs.pop('name', None) or \
+        (getattr(layer, 'name', None) and layer.name + '_bn')
```

Comment:
Looks good except that we usually do a double indent here. Or use visual indent with an extra pair of brackets:
```python
bn_name = (kwargs.pop('name', None) or
           getattr(layer, 'name', None) and layer.name + '_bn')
```
Sorry for nitpicking! It's just that a single indentation level looks like an indented code block rather than a continued line.